src/main.rs

```diff
@@ -39,7 +39,7 @@ use util::*;
 mod prelude;
 
 #[group]
-#[commands(ping, about, serverinfo, botsuggest, privacy, servers)]
+#[commands(ping, about, serverinfo, botsuggest, privacy, servers, teamstats,leaderboard)]
```

Comment:
Maybe split this into a separate command group

Reply:
If I split it into another group does it have to be in another file or can it stay in general.rs or I suppose the better question if it can stay is should it as far as code organization is concerned?

Reply:
For code organization it should be in a separate file (groupname.rs)